cocos/2d/CCLabelBMFont.h

```diff
@@ -42,12 +42,6 @@ Use any of these editors to generate BMFonts:
 
 NS_CC_BEGIN
 
-#if defined(__GNUC__) && ((__GNUC__ >= 4) || ((__GNUC__ == 3) && (__GNUC_MINOR__ >= 1)))
-#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
-#elif _MSC_VER >= 1400 //vs 2005 or higher
-#pragma warning (push)
-#pragma warning (disable: 4996)
```

Comment:
@ricardoquesada We should remove 'CCLabelBMFont.h'/'CCLabelTTF.h' from 'cocos2d.h' if not hidden this warn.